R/eval-walk.R

```diff
@@ -18,36 +18,12 @@ vars_select_eval <- function(vars,
     return(pos)
   }
 
-  uniquely_named <- uniquely_named %||% is.data.frame(data)
-
-  if (!is_symbolic(wrapped)) {
```

Comment:
I removed this early exit for literals so that all inputs go through the same path via the tidyselect interpreter.

Reply:
Seems like a very good idea for consistency